package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "bg",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "stylecss": "stylelint './components/**/*.js'"
```

Maybe let's add a script for linting js and linting both css and js
```suggestion
    "lint:css": "stylelint './**/*.js'",
    "lint:js": "eslint './**/*.js'",
    "lint": "npm run lint:js && npm run lint:css"
```